src/index.js

```diff
@@ -48,7 +48,7 @@ import isHSL from './lib/isHSL';
 
 import isISRC from './lib/isISRC';
 
-import isIBAN from './lib/isIBAN';
+import isIBAN, { locales as ibanCountryCodes } from './lib/isIBAN';
```

Comment:
```suggestion
import isIBAN, { locales as isIBANLocales } from './lib/isIBAN';
```
I would name it `isIBANLocales` to respect existing naming.

Reply:
@fedeci changed 😄